_src/_includes/themes/apache/home.html

```diff
@@ -79,15 +80,15 @@
 </section>
 
 <!-- Awards -->
-<section class="full-stripe full-stripe--alternate-2">
+<!-- <section class="full-stripe full-stripe--alternate-2">
```

Comment:
Please don't hide this. @Deron was just saying that with this PR, the VLDB paper award image for this section had accidentally been replaced by the Apache logo. Looking below, it looks like the `award` style id was changed, but it's still being used by both sections still.